config_hub/finetune/llama-2-7b/full.yaml

```diff
@@ -1,39 +1,96 @@
+
+# The path to the base model checkpoint directory to load for finetuning. (type: <class 'Path'>, default: checkpoints/stabilityai/stablelm-base-alpha-3b)
+checkpoint_dir: checkpoints/meta-llama/Llama-2-7b-hf
+
+# Where to save checkpoints and logs. (type: <class 'Path'>, default: out/finetune/full)
+out_dir: out/finetune/full-llama2-7b
+
+# The precision to use for finetuning. Possible choices: "bf16-true", "bf16-mixed", "32-true". (type: Optional[str], default: null)
```

Comment:
```suggestion
# The precision setting for finetuning the model weights. Possible choices: "bf16-true", "bf16-mixed", "32-true". (type: Optional[str], default: null)
```

Reply:
This one I am not sure. Mixed precision training is not just about the weights. I think we know what is meant by saying "precision for finetuning"? We could have extensive docs about which setting applies to what somewhere else maybe.